main.css

```diff
@@ -0,0 +1,29 @@
+#main{
+    text-align: center;
+    background: #eee;
+}
+#img-div{
+    background: white;
```

Comment:
Using color names is considered bad practice. I suggest you change the color to its rgb or hex equivalent